test/linter/test-browsers.js

```diff
@@ -46,7 +46,7 @@ const browsers = {
  * @param {string[]} displayBrowsers
  * @param {string[]} requiredBrowsers
  * @param {string} category
- * @param {{error:function(...unknown):void}} logger
+ * @param {import('../utils').Logger} logger
```

Comment:
Is this in preparation to have a logger in utils? I only see a comment about `logger` in utils.js but no implementation. Can you elaborate?

Reply:
That’s because the logger is just an object with a defined structure, like an interface in TypeScript.

Reply:
Ah okay, cool. It looks like it makes sense. Thanks :)